cli/cmd/inject.go

```diff
@@ -412,6 +412,10 @@ func getOverrideAnnotations(values *charts.Values, base *charts.Values) map[stri
 		overrideAnnotations[k8s.ProxyOpaquePortsAnnotation] = proxy.OpaquePorts
 	}
 
+	if proxy.PodInboundPorts != baseProxy.PodInboundPorts {
+		overrideAnnotations[k8s.ProxyOutboundPortAnnotation] = proxy.PodInboundPorts
```

Comment:
Can you add a testcase to inject_test.go so that we can confirm the behaviour through CLI flag?

Reply:
Good catch! Although it turns out this line is not doing anything since there is no flag corresponding to this new annotation. And probably not a good idea to add a new flag for this, which is provided as an escape hatch for an edge case. So I'll just delete these lines.

Reply:
Makes sense!